manager/orchestrator/replicated/task_reaper_test.go

```diff
@@ -493,11 +493,9 @@ func TestServiceRemoveDeadTasks(t *testing.T) {
 
 	// Set both task states to RUNNING.
 	updatedTask1 := observedTask1.Copy()
-	updatedTask1.DesiredState = api.TaskStateRunning
```

Comment:
@anshulpundir sorry I don't follow why this is removed?

Reply:
because desired state is  api.TaskStateRunning when a task is created.

Reply:
Okay, makes sense.